src/ui/actions/comments.ts

```diff
@@ -155,20 +155,23 @@ export function createLabels(
   return async (dispatch, getState, { ThreadFront, replayClient }) => {
     await ThreadFront.ensureAllSources();
     const state = getState();
-    const sourceId = handleUnstableSourceIds(sourceLocation.sourceUrl, state);
 
-    if (sourceId) {
-      sourceLocation = {
-        ...sourceLocation,
-        sourceId,
-      };
-    }
+    const { line, sourceUrl } = sourceLocation;
 
-    const { sourceUrl, line } = sourceLocation;
+    let sourceId: string | null = sourceLocation.sourceId || null;
 
     const filename = sourceUrl ? getFilenameFromURL(sourceUrl) : "unknown source";
+
     if (!sourceId) {
```

Comment:
I'm not sure why we weren't already doing this check. Was there a good reason, or was this an oversight?

It seems analogous to this code that @jcmorrow recently changed:
https://github.com/replayio/devtools/blob/79d06aab168237d525f40c25d068f92ef7c16001/src/devtools/client/debugger/src/actions/sources/select.ts#L138-L142

Am I missing something?

Reply:
I haven't poked at this code myself, but the diff _seems_ valid...

Reply:
I'm not 100% sure I understand the change here, but it seems worth pointing out that even if we have an ID that doesn't mean it's the *correct* ID.

Reply:
Thanks for that added context, Josh. I'll make that change before this lands then.

Reply:
I think I've adjusted things now as discussed.